src/device/device-management.spec.ts

```diff
@@ -254,4 +255,53 @@ describe('Device Management', () => {
       expect(localDisplayStream.contentHint).toBe('motion');
     });
   });
+
+  describe('getDevices', () => {
```

Comment:
I'd also suggest covering cases with an error if we add `getDevices` tests. It's useful to understand how it should work, not only with the happy flow.